content/en/docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -38,7 +38,8 @@ Choose an IP range that is *NOT* part of the public IP address range.
 Add the configuration to the etcd server on fed-master.
 
 ```shell
-etcdctl set /coreos.com/network/config < flannel-config.json
+if the etcd server version 3.x, and as flanneld does not supoort etcd v3.x yet
```

Comment:
line 41 doesn't belong to the code snippet section.

Reply:
s/supoort/support